client/src/ts/readouts.ts

```diff
@@ -0,0 +1,101 @@
+// Module that builds a readout for the driver display containing a value and
```

Comment:
Would it make sense to rename this file `readout.ts`? You named the other one `dial.ts`